panel/src/components/Lab/PlaygroundView.vue

```diff
@@ -9,9 +9,16 @@
 		</k-header>
 		<k-tabs :tab="tab" :tabs="tabs" />
 
-		<component :is="component" v-if="component" v-bind="props" />
-		<!-- eslint-disable-next-line vue/no-v-html, vue/no-v-text-v-html-on-component -->
-		<component :is="'style'" v-if="styles" v-html="styles" />
+		<template v-if="compiler === false">
```

Comment:
`Question:` Is there an extra benefit of the template tags over putting the `v-if` directly on the `k-box`?

Reply:
No, there isn't. I don't know why I prefer it sometimes. It just happens :)

Reply:
😄 I'll allow it 😆 was just curious